data/core/terrain-graphics/new-macros.cfg

```diff
@@ -3529,7 +3529,7 @@ wall#endarg
             type={TERRAINLIST}
         [/tile]
 
-#        rotations=bl
+        #        rotations=bl
```

Comment:
Here's another case where there's excessive whitespace _after_ the comment character. (Or rather, soneone added a comment character before the indentation rather than after it.)

Reply:
well anyways the `wmlindent` parts are done now